modules/shared/src/main/scala/org/tessellation/schema/address.scala

```diff
@@ -56,6 +59,9 @@ object address {
 
     implicit val orderDAGAddress: Order[DAGAddress] =
       orderOf[String, DAGAddressRefined]
+
+    implicit val orderingDAGAddress: Ordering[DAGAddress] =
```

Comment:
Can't you just convert the order above to an ordering?

Reply:
Yep, I can. Fixing it.